pages/learning.tsx

```diff
@@ -21,8 +21,8 @@ const LearningPage: NextPage = () => (
           <Heading fontFamily="body" fontWeight="600" as="h2" textTransform="capitalize" fontSize={48} textAlign="center">
```

Comment:
```suggestion
          <Heading fontFamily="body" fontWeight="600" as="h2" fontSize={48} textAlign="center">
```

Remove this one as well.

Reply:
done sir

Reply:
there is a conflict about the correct use of the word "di"
<img width="733" alt="Screen Shot 2020-10-31 at 19 32 39" src="https://user-images.githubusercontent.com/26411157/97778150-21c9cf00-1bb0-11eb-8019-0adec296124b.png">